packages/ui-components/test/components/test-NatAmountInput.js

```diff
@@ -0,0 +1,190 @@
+// @ts-check
+
+import '@agoric/install-ses';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import test from 'ava';
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { shallow, render } from 'enzyme';
+
+// @ts-ignore path is correct for compiled output
+import { NatAmountInput } from '../../../dist'; // eslint-disable-line import/no-unresolved
```

Comment:
Can you please try this out?  With the `"main"` change in `package.json` this should always work.

```suggestion
import { NatAmountInput } from '../../..';
```

Reply:
I don't think this will work, because the source needs to be compiled for the tests to run. I am compiling manually through babel.

Reply:
Tested and confirmed that the compiled version is necessary because of the JSX:

```
 Uncaught exception in compiled/test/components/test-NatAmountInput.js

  /Users/katesills/code/agoric-sdk/packages/ui-components/src/components/NatAmountInput.js:31
      <TextField
      ^

  SyntaxError: Invalid or unexpected token

```